backend/src/controllers/tasks.controller.ts

```diff
@@ -28,6 +29,32 @@ export class TasksController {
     return this.tasksService.insertTask(task);
   }
 
+  @Post('/new-comment/:taskId')
+  async insertComment(
+    @Param('taskId') taskId: number,
+    @Body() comment: Comment,
+  ): Promise<void> {
+    if (!comment.comment) {
+      console.log('Comment is missing.');
+      return;
+    }
+    if (comment.comment.length > 255) {
+      console.log('Comment is too long.');
+      return;
+    }
+    const task = await this.tasksService.tasksRepository.findOne({
+      where: { id: taskId },
+    });
+    if (!task) {
+      console.log('Invalid task id.');
+      return;
```

Comment:
In this case, can we return a `404 Not Found` to make it more clear to the clients?